src/xdsl/dialects/llvm.py

```diff
@@ -4,8 +4,10 @@
 from xdsl.irdl import (ParameterDef, AnyAttr, irdl_op_builder,
                        irdl_attr_definition, AttributeDef, OperandDef,
                        ResultDef, irdl_op_definition, builder)
-from xdsl.ir import MLContext, ParametrizedAttribute, TYPE_CHECKING, Attribute, Operation
-from xdsl.dialects.builtin import StringAttr, ArrayOfConstraint, ArrayAttr
+from xdsl.ir import (MLContext, ParametrizedAttribute, TYPE_CHECKING,
```

Comment:
My bad, I never realized that `TYPE_CHECKING` was imported from `xdsl.ir` previously.
Could you import it from `typing` instead?